src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitSender.java

```diff
@@ -27,55 +26,77 @@
 import com.exactpro.th2.common.schema.message.MessageSender;
 import com.exactpro.th2.common.schema.message.impl.rabbitmq.connection.ConnectionManager;
 
+import static com.exactpro.th2.common.metrics.CommonMetrics.EXCHANGE_LABEL;
+import static com.exactpro.th2.common.metrics.CommonMetrics.ROUTING_KEY_LABEL;
+import static com.exactpro.th2.common.metrics.CommonMetrics.TH2_PIN_LABEL;
+import static com.exactpro.th2.common.metrics.CommonMetrics.TH2_TYPE_LABEL;
 import io.prometheus.client.Counter;
+import static java.util.Objects.requireNonNull;
 
 public abstract class AbstractRabbitSender<T> implements MessageSender<T> {
     private static final Logger LOGGER = LoggerFactory.getLogger(AbstractRabbitSender.class);
 
-    private final AtomicReference<String> sendQueue = new AtomicReference<>();
+    private static final Counter MESSAGE_SIZE_PUBLISH_BYTES = Counter.build()
+            .name("th2_rabbitmq_message_size_publish_bytes")
+            .labelNames(TH2_PIN_LABEL, TH2_TYPE_LABEL, EXCHANGE_LABEL, ROUTING_KEY_LABEL)
+            .help("Quantity of published message bytes to RabbitMQ. " +
+                    "The message is meant any message transferred via RabbitMQ, it can contain th2 batch message or event or other content")
```

Comment:
```suggestion
                    "The message is meant for any message transferred via RabbitMQ, it may contain th2 batch message or event or other content")
```